.circleci/config.yml

```diff
@@ -37,6 +37,7 @@ jobs:
     environment:
       - BINARY_PATH: "/home/circleci/project/builds/gaia/linux_amd64/gaiacli"
       - NODE_BINARY_PATH: "/home/circleci/project/builds/gaia/linux_amd64/gaiad"
+      - CI: "true"
```

Comment:
This is already [set automatically by CircleCI](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables).